### ISSUE TITLE
Alertes Mattermost CNPG

### ISSUE THREAD

**Issue by igorrenquin** [SocialGouv/support#391]

**Comment by octomir** [comment]
@revolunet est-il possible de recetter le ticket et le fermer 
merci

**Comment by octomir** [comment]
des alertes sont effectifs sur le canal mattermost un test sur la suppression d'un replica  doit être fait

**Comment by revolunet** [comment]
Comment tester la suppression d'un replica ?

**Comment by revolunet** [comment]
maybe related : https://github.com/cloudnative-pg/cloudnative-pg/issues/1814

**Comment by achauve** [comment]
yes je suis tombé dessus en cherchant aussi, mais pas de solution claire ? effectivement passer la métrique sur le primary alors qu'on regarde le lag sur les replica ça paraît louche ?

**Comment by revolunet** [comment]
Règles par défaut : https://github.com/cloudnative-pg/charts/blob/8fe6dee4e7966ca474d33f0769d2207a38befb96/charts/cloudnative-pg/values.yaml

**Comment by LucasBassoOcto** [comment]
Session de pair afin de parametrer le monitoring cnpg (délai entre le réplica et le primaire) dans la chart helm de l'operateur

**Comment by igorrenquin** [comment]
TO DO :

SRE

- [ ] Analyse des alertes générées par CNPG
  - replicas
  - other to find

OPS

- [ ] Appliquer le fix pour empécher le floud

**Comment by revolunet** [comment]
## Erreurs PGReplication

Comportement normal sur les alertes vérifiées : les données sont MAJ toutes les heures seulement sur ces DBs

**Comment by igorrenquin** [comment]
Ajout d'un filtre sur la PR de julien. 
Cause de l'erreur : la requête de la métrique s'applique sur tous le cluster CNPG plutôt que sur le noeud master. 
En attente d'une PR du côté CNPG (nouvelle realease)
